docs/source/getting_started/quickstart.rst

```diff
@@ -30,9 +30,12 @@ Wait for few minutes and run `kubectl get pods -n aibrix-system` to check pod st
     aibrix-redis-master-84769768cb-fsq45         1/1     Running   0          5h24m
 
 
-Deploy base model
+Deploy Base Model
 ^^^^^^^^^^^^^^^^^
 
+.. note::
+In the given example, please ensure that your node is equipped with NVIDIA GPUs. Otherwise, the pod will stuck in pending state.
```

Comment:
Most users will expect to run this on GPU env. I think here we do not want to implicitly mention NVIDIA GPUs is needed. Technically, amd gpus or other NPUs works as well. we can say [device plugin](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/) for accelerator is needed.